lib/class-wp-theme-json-gutenberg.php

```diff
@@ -144,7 +144,7 @@ class WP_Theme_JSON_Gutenberg {
 			'path'              => array( 'color', 'duotone' ),
 			'prevent_override'  => array( 'color', 'defaultDuotone' ),
 			'use_default_names' => false,
-			'value_func'        => 'gutenberg_get_duotone_filter_property',
+			'value_func'        => null, // Don't output CSS Custom Properties for duotone.
 			'css_vars'          => '--wp--preset--duotone--$slug',
```

Comment:
```suggestion
			'value_func'        => null, // CSS Custom Properties for duotone are handled by block supports in class-wp-duotone-gutenberg.php.
			'css_vars'          => null,
```

@oandregal correct me if I'm wrong about how this should be handled.

Reply:
This will probably result in some unused code paths in global styles, but as this is already a pretty large change to review the plan is to investigate that in a follow-up.

Reply:
I assume we'll add a new issue for the follow-up to the [Duotone filters](https://github.com/WordPress/gutenberg/issues/47304) tracking issue?

Reply:
Yeah, I'll be updating it after this is merged with any follow-ups that we may need.